examples/tf/ppo_pendulum.py

```diff
@@ -39,7 +39,7 @@ def run_task(*_):
         max_path_length=100,
         n_itr=488,
         discount=0.99,
-        step_size=0.01,
+        loss_clip_range=0.01,
```

Comment:
I've separated `loss_clip_range` and `max_kl_step`. Theano/NPO still uses `step_size`, but I think we can leave it there. 
There is a problem when I change the corresponding params in tests/benchmarks/test_benchmark_*.py. The pylint raises the following errors:
```
hookid: pylint

************* Module tests.benchmarks.test_benchmark_ppo
tests/benchmarks/test_benchmark_ppo.py:E0401: 14,0: Unable to import 'baselines.bench'
tests/benchmarks/test_benchmark_ppo.py:E0401: 16,0: Unable to import 'baselines.bench'
tests/benchmarks/test_benchmark_ppo.py:E0401: 18,0: Unable to import 'baselines.common'
tests/benchmarks/test_benchmark_ppo.py:E0401: 19,0: Unable to import 'baselines.common.vec_env.dummy_vec_env'
tests/benchmarks/test_benchmark_ppo.py:E0401: 20,0: Unable to import 'baselines.common.vec_env.vec_normalize'
tests/benchmarks/test_benchmark_ppo.py:E0401: 21,0: Unable to import 'baselines.logger'
tests/benchmarks/test_benchmark_ppo.py:E0401: 22,0: Unable to import 'baselines.ppo2'
tests/benchmarks/test_benchmark_ppo.py:E0401: 23,0: Unable to import 'baselines.ppo2.policies'
tests/benchmarks/test_benchmark_ppo.py:W0611: 17,0: Unused common imported from baselines
************* Module tests.benchmarks.test_benchmark_trpo
tests/benchmarks/test_benchmark_trpo.py:E0401: 15,0: Unable to import 'baselines.bench'
tests/benchmarks/test_benchmark_trpo.py:E0401: 16,0: Unable to import 'baselines.common.cmd_util'
tests/benchmarks/test_benchmark_trpo.py:E0401: 17,0: Unable to import 'baselines.common.tf_util'
tests/benchmarks/test_benchmark_trpo.py:E0401: 18,0: Unable to import 'baselines.ppo1.mlp_policy'
tests/benchmarks/test_benchmark_trpo.py:E0401: 19,0: Unable to import 'baselines.trpo_mpi'
```
Anybody have ideas why?

Reply:
this is because pylint is now checking whether all of your imports are actually importable from the default garage install.

are you sure that those baselines modules exist in the default environment.yml? the packages changed and it needs updating?